### DISCUSSION TITLE
Autoupdate suggestion

### DISCUSSION THREAD

**Discussion by dzhonzojdberg** [arriven/db1000n#500] [Ideas]
Hi, I have a suggestion to create a launcher which at startup and also with a certain frequency during execution will check the relevance of the version and build or download update it if necessary.

**Comment by arriven** [comment]
I've been thinking about that and I must say that it's a great idea. I wouldn't work on it myself but having a GUI wrapper would also allow adding more neat features like monitoring system resources and scaling the performance accordingly and would probably allow me to focus on actual attacks instead of user friendly features.

Note that there's already some experimental auto update functionality present but it's not being tested (I have additional security hardening on my machine that just breaks this feature).

**Reply by dzhonzojdberg** [reply]
You say very clever things, frankly and logically. Thank you for explaining the problem. I would like to make a more significant contribution, but I lack programming skills, but I will try to involve people who can. Whatever it is, remember that you are not alone and do a great job. Thank you!
